community/contributing/nightly-builds/index.md

```diff
@@ -150,3 +149,28 @@ All processes are run as the `jenkins` user and most of the useful links are in
 ```bash
 $ sudo su - jenkins
 ```
+
+### Common Build Failures
+
+#### Bintray Conflicts
+
+If an artifact is uploaded to the Bintray repository but never published
+(either because of a transient Bintray error or an interrupted build), you'll
+get an error like this:
+
+> Bintray API Request 'create version 0.20.0-20200512192702' failed with HTTP response 409 Conflict
+
+Delete the artifact to resolve the issue.
+
+First, navigate to the specific version [in the Bintray
```

Comment:
Can you convert these last paragraphs into steps?

Reply:
Done, much better, thanks!